lib/src/git_backend.rs

```diff
@@ -1218,6 +1241,13 @@ impl Backend for GitBackend {
         }
         let extras = serialize_extras(&contents);
 
+        if self.change_id_setting {
+            extra_headers.push((
+                BString::new(CHANGE_ID_COMMIT_HEADER.to_vec()),
+                BString::new(contents.change_id.reverse_hex().into()),
+            ));
+        }
```

Comment:
nit: move this before `let extras` so it's together with the other `extra_headers` code?